lib/output/aws_s3.go

```diff
@@ -123,6 +144,20 @@ calculated per message of a batch.
 
 Metadata fields on messages will be sent as headers, in order to mutate these values (or remove them) check out the [metadata docs](/docs/configuration/metadata).
 
+### Tags
+
+Tag fields will be stored as object tags against the S3 object. For example:
+
+` + "```yaml" + `
+output:
+	aws_s3:
```

Comment:
Looks like your IDE has mixed tabs and spaces, mine does this all the time and it drives me crazy embedding yaml configs in a Go string literal.

Reply:
And lo! My changes seem to have no effect, I'll spend some more time on this tomorrow 😄

Reply:
Righto, that's fixed!